test/unit_test/rag/nlp/test_gaussdb_retrieval.py

```
@@ -0,0 +1,528 @@
+    memory_module = memory
+    saved_memory_utils = getattr(memory_module, "utils", None) if memory_module else None
+
+    settings_stub = _install_module(
+        "common.settings",
+        DOC_ENGINE_GAUSSDB=True,
+        DOC_ENGINE_INFINITY=False,
+        DOC_ENGINE_OCEANBASE=False,
+        DOC_ENGINE_SERENEDB=False,
+    )
+    if common_module is not None:
+        setattr(common_module, "settings", settings_stub)
```

Ruff (0.16.1)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)